src/main/resources/lang/fr_fr.json

```diff
@@ -55,6 +55,9 @@
 	"popup_menu.forward": "Restaurer",
 	"popup_menu.mark_deobfuscated": "Marquer comme déobfusqué",
 	"popup_menu.reset_obfuscated": "Réinitialiser à obfusqué",
+	"popup_menu.zoom.in": "Zoom in",
+	"popup_menu.zoom.out": "Zoom out",
+	"popup_menu.zoom.reset": "Reset zoom",
```

Comment:
`Agrandir`, `Rapetisser`, `Réinitialiser la grandeur`

Reply:
Or maybe @Yanis48 do you have some better suggestions?

Reply:
Zoomer, Dézoomer, Réinitialiser le zoom